tests-behat/revealEye.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Password with revealEye
+
+  Scenario:
+    Given I am on "form-control/input2.php"
+    When I fill in "password_re_norm" with "123"
+    Then "password_re_norm" should contain text "***"
```

Comment:
A little unrelated - you found a bug with Behat IMO - missing step should emit non-zero exit code. It might be worth to investigate and report to Behat/Mink. Currently, only the "lowest" test pipelines can detect such issue correctly and fail the CI.

Reply:
I have no clue at all on what you wrote. I'll take it at face value.
I'm not even sure the script works as I don't know if the field returns "***" to behat?

I wouldn't even know where to start with trying to report a bug in this case... :)

Reply:
I do not know where to report it too ATM :)

Exit code - CLI exit code returned of the Behat CLI application

> I'm not even sure the script works as I don't know if the field returns "***" to behat?

Behat detected some grammar that you have written but not recognized.

It takes some time to understand Behat/Mink, that is normal. In linux, you can setup the testing environment quite easily, in Windows, it is a little more complicated as you need to convince all the drivers to work with your Chrome browser.

Reply:
As a matter of fact, I run only Linux... Have done since 1992... I just never automated tests...

Reply:
That saves you a lot of troubles - see https://github.com/atk4/ui/blob/5.2.0/.github/workflows/test-unit.yml#L370-L378 - with more or less equivalent code you should be able to run the Behat on your local computer. The run command itself is a few lines below - https://github.com/atk4/ui/blob/5.2.0/.github/workflows/test-unit.yml#L383-L384

Reply:
Ok, I'll check it out...